protocol/activity_center.go

```diff
@@ -44,3 +47,23 @@ func (n *ActivityCenterNotification) Valid() error {
 	return nil
 
 }
+
+func showMentionActivityCenterNotification(publicKey ecdsa.PublicKey, message *common.Message, chat *Chat, responseTo *common.Message) bool {
+	if chat != nil && !chat.Active {
+		return false
+	}
+
+	publicKeyString := common.PubkeyToHex(&publicKey)
+	mentioned := false
+	for _, mention := range message.Mentions {
```

Comment:
We changed the code, I believe now you can just check for:
`message.Mentioned`

Reply:
Good to know, will change it :)